pkg/server/routes.go

```diff
@@ -53,20 +53,16 @@ func (srv *RegistrationServer) SetupRoutes() error {
 
 		// /status is something you should always have in any of your services,
 		// please leave it as is.
-		healthService := health.New(srv.logger, srv.Config())
-		// TODO uncomment these once the services are available
-		//signupService := signup.NewSignupService(srv.logger, srv.Config())
-		//signupCallbackService := signup.NewSignupCallbackService(srv.logger, srv.Config())
```

Comment:
Can we leave these lines here (commented out) until we are ready to implement them?  I am working on this endpoint this week.

Reply:
we no longer need the signupCallbackService, see other comments for details